.dockerignore

```diff
@@ -0,0 +1,3 @@
+dist
+node_modules
```

Comment:
I added node_modules here because this makes it a lot faster to push the context to the docker daemon.
I also realized that yarn was anyway redownloading everything :/ Maybe because its cache was absent? It's inconsistent with how it works in CircleCI otherwise, where we cache `node_modules` between jobs, and there yarn doesn't redownload...
I couldn't figure it why this difference, and instead I decided to go this way, which works actually quite fast.